packages/react-table/src/components/Table/utils/transformers.test.tsx

```diff
@@ -144,54 +78,13 @@ describe('Transformer functions', () => {
       const column = { extraParams: { sortBy: { index: 0, direction: 'desc' }, onSort } };
       const returnedData = sortable('', { column, columnIndex: 0 } as IExtra);
       expect(returnedData).toMatchObject({ className: 'pf-c-table__sort pf-m-selected' });
-      const view = mount(returnedData.children as React.ReactElement<any>);
-      view.find('button').simulate('click');
+
+      render(returnedData.children as React.ReactElement<any>);
+      userEvent.click(screen.getByRole('button'));
       expect(onSort.mock.calls).toHaveLength(1);
     });
   });
 
-  test('simpleCellActions', () => {
-    const actions: IActions = [
-      {
-        title: 'Some',
-        onClick: jest.fn()
-      }
-    ];
-
-    const actionConfigs = [
-      {
-        actions: [] as IActions
-      },
-      {
-        actions
-      },
-      {
-        actionResolver: () => null as (IAction | ISeparator)[]
-      },
-      {
-        actionResolver: () => actions as (IAction | ISeparator)[]
-      },
-      {
-        actionResolver: () => actions as (IAction | ISeparator)[],
-        areActionsDisabled: () => false
-      },
-      {
-        actions,
-        rowData: {
-          disableActions: true
-        } as IRowData,
-        expectDisabled: true
-      },
-      {
-        actionResolver: () => actions,
-        areActionsDisabled: () => true,
-        expectDisabled: true
-      }
-    ];
-
-    actionConfigs.forEach(testCellActions);
```

Comment:
The actions are dynamically built in the table, so testing that they are built and firing the right events at the right times is probably something we should keep.

Reply:
None of these iterated tests in `transformers` made much sense to me, so I can take another look, and I recall trying to get these to work, but might be something I make a separate story in our backlog for to address later.

Reply:
that'd be fine if you end up doing that i think